src/ApiTestCase.php

```diff
@@ -228,37 +214,100 @@ private function getCalledClassFolder()
         $calledClass =  get_called_class();
         $calledClassFolder = dirname((new \ReflectionClass($calledClass))->getFileName());
 
-        $this->folderExists($calledClassFolder);
+        $this->directoryCheck($calledClassFolder);
 
         return $calledClassFolder;
     }
 
+    protected function loadAllFixtures()
+    {
+        $this->loadFixturesByName();
+    }
+
+    protected function loadFixturesByName(array $fixturesNames = array())
+    {
+        $loader = new Loader();
+
+        $directory = $this->getFixturesFolder();
+        $this->directoryCheck($directory);
+        $paths = $this->getFixturePathsFromDirectory($directory, $fixturesNames);
+
+        foreach ($paths as $path) {
+            $objects = $loader->load($path);
+            $this->persistObjects($objects);
+        }
+
+        $this->entityManager->flush();
+    }
+
     /**
      * @param string $directory
+     * @param array $fixtureNames
      *
      * @return array
      */
-    private function getFilesFromDirectory($directory)
+    private function getFixturePathsFromDirectory($directory, array $fixtureNames = array())
     {
-        $this->folderExists($directory);
+        $fileNames = $this->loadFileNamesFromDirectory($directory);
+        $filePaths = array();
 
-        $files = scandir($directory);
-        $files = array_diff($files, array('..', '.'));
-        if (!$files) {
-            throw new \RuntimeException(sprintf('There is no files to load in folder %s', $directory));
+        foreach ($fileNames as $fileName) {
+            $filePath = $directory.'/'.$fileName;
+
+            if (is_dir($filePath)) {
+                $filePaths = array_merge($filePaths, $this->getFixturePathsFromDirectory($filePath, $fixtureNames));
+            } elseif ($this->isExpectedFixture($fileName, $fixtureNames)) {
+                $filePaths[] = $filePath;
+            }
         }
 
-        return $files;
+        return $filePaths;
+    }
+
+    /**
+     * @param string $name
+     * @param array $fixtureNames
+     *
+     * @return bool
+     */
+    private function isExpectedFixture($name, array $fixtureNames = array())
+    {
+        if (empty($fixtureNames) || in_array($name, $fixtureNames)) {
```

Comment:
Use `true` as third argument of `in_array` to use strict comparison, loose one won't work as you expect :)